### DISCUSSION TITLE
0.45.0 "$" character in character module

### DISCUSSION THREAD

**Discussion by AppleGamer22** [starship/starship#1694] [Q&A] [answered]
After updating version 0.45.0, I have not managed to put a "$" character in my prompt character without raising a parsing error.

Is there a way to do so?

**Comment by AppleGamer22** [comment]
* I found a workaround which involves using the fullwidth dollar sign symbol `\uFF04`:
```toml
success_symbol = "[\uFF04](bold green)"
```
* It is slightly different than the default dollar sign character `\u0024`.

**Comment by ghost** [comment, marked as answer]
You should be able to do it like this:
```toml
success_symbol = "[\\$](bold green)"
```

Which escapes the dollar sign.

**Reply by AppleGamer22** [reply]
I have just learned something new.

Thanks a lot.
